build/build.sh

```diff
@@ -0,0 +1,192 @@
+#!/bin/bash
+set -euo pipefail
+IFS=$'\n\t'
+# 
+# LiskHQ/lisk-build
+# Copyright (C) 2017 Lisk Foundation
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+######################################################################
```

Comment:
We use different license header in Lisk Core, please copy same from any other file.